articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+author: Andy Webb
+toc_rootlink: Service Scope
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: UKCloud for Microsoft Azure Service Scope
+toc_fullpath: Service Scope/asz-sco.md
+toc_mdlink: asz-sco.md
+---
+
+# UKCloud for Microsoft Azure Service Scope
+
+## About this document
+
+This document describes the boundaries of the UKCloud for Microsoft Azure service, along with the division of responsibilities between UKCloud and the customer, to facilitate the provisioning and ongoing use of the service.
+
+## About UKCloud for Microsoft Azure
+
+UKCloud for Microsoft Azure harnesses the innovation of Microsoft’s Azure platform and we deliver this service from within our award-winning government-grade multi-cloud environment – addressing the service, connectivity and diversity requirements of UK public sector organisations and their industry partners. The service is based on the Microsoft Azure Stack product, which has been developed by Microsoft as an extension of the Azure public cloud.
+
+## Service options
+
+You can build a tailored solution to meet your requirements by mixing and matching service options. The choice does not lock you to one service or size, allowing you to scale resources up or down as your requirements change.
+
+By default customers will initially have access to:
+
+- XX vCPU
+- YY GB memory
+- ZZ GB storage
+
+The options below are all currently available within the service:
+
+## IaaS Compute 
+Virtual machines (VMs) provide the basic building block providing compute resource within UKCloud for Microsoft Azure. Each standard machine has a pre-allocated number of vCPU's, amount of memory and a temporary disk. The available sizes and families of VM's are contiually increasing, broadly, the following machine families are supported:
+
+- Basic and Standard A-series. Have CPU performance and memory configurations best suited for entry level workloads like development and test.  
+- Standard D and DS Series. Have optimised CPU-to-memory configuration making them suitable for most general-purpose applications. A subset of the D-series VMs also sport higher memory per CPU making them suitable for applications that require higher amounts of memory. 
+- Basic and Standard F series. Are optimised for CPU intensive workloads so have a higher CPU to memory configurations.
+
+The full list of supported machines can be found [here](https://docs.microsoft.com/en-gb/azure/azure-stack/user/azure-stack-vm-sizes).
+
+Customers can deploy VMs using the UKCloud for Microsoft Azure portal and other tooling.
+
+> [!Warning]
+> Each VM contains a temporary disk. The temporary disk provides short-term storage for applications and processes and is intended to only store data such as page or swap files. Data on the temporary disk may be lost during a maintenance event or when you redeploy a VM. 
+
+## Storage Options 
+There are several ways in which storage can be addressed in UKCloud for Microsoft Azure:
+
+- Azure blob storage (Page)
```

Comment:
Add carriage return after this line and indent the next line with a tab so that indentation works correctly (same for other bullets in this list)